modules/Json.tla

```diff
@@ -8,29 +8,40 @@ LOCAL INSTANCE TLC
 
 -----------------------------------------------------------------------------
 
-LOCAL ToJsonKeyValue(F, d) == 
-  ToString(ToString(d)) \o ":" \o ToString(F[d])
-
-RECURSIVE ToJson(_,_)
-LOCAL ToJson(F, D) == \* LOCAL just a hint for humans.
-  LET d == CHOOSE d \in D: TRUE
-  IN IF D = DOMAIN F
-     THEN "{" \o ToJsonKeyValue(F, d) \o IF D \ {d} = {}
-                                         THEN "}"
-                                         ELSE ToJson(F, D \ {d})
-     ELSE ", " \o ToJsonKeyValue(F, d) \o IF D \ {d} = {}
-                                          THEN "}" 
-                                          ELSE ToJson(F, D \ {d})
+ToJson(value) ==
+  (*************************************************************************)
+  (* Converts the given value to a JSON string. Records are converted to   *)
+  (* JSON objects, tuples to JSON arrays, and scalar values as their JSON  *)
+  (* representation.                                                       *)
+  (*************************************************************************)
+  ""
```

Comment:
If would be nice to have (high-level) definitions of all operators to document what they do (no need to actual evaluate to the same value as the module override).  See e.g. definition of `JavaTime` at https://github.com/tlaplus/Examples/blob/536628fc9ec15263f17e5849f0a9a8e2fa20241d/specifications/SpecifyingSystems/TLC/TLC.tla#L8

Reply:
Okay that makes sense